framework/src/application/storage/migrations/node/20200316000000_add_new_dpos_to_mem_accounts.sql

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright © 2020 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+
+ALTER TABLE "mem_accounts" ADD COLUMN IF NOT EXISTS "totalVotesReceived" BIGINT NOT NULL DEFAULT 0;
```

Comment:
Jut for reference `BIGINT` in PostgreSQL and `bigint` in NodeJS has different ranges. DId we take this point in consideration?

Reply:
yes, and to make the efficient order, I think it has to be BIGINT is the current option (same as voteWight now